.github/workflows/publish.yml

```
@@ -9,54 +9,44 @@ on:
     - cron: '31 13 * * 3'
 
 jobs:
-  Build:
+  Version:
     outputs:
       created: ${{ steps.daily-version.outputs.created }}
+      version: ${{ steps.daily-version.outputs.version }}
```

```suggestion
      created: ${{ steps.daily-version.outputs.created }}
      version: ${{ steps.daily-version.outputs.version }}
```

I wonder how these 2 can be merged. Perhaps the action can output a `newVersion` only if it just created one.